docs-conceptual/az-0.3.0/new-azureps-module-az.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Introducing the Azure PowerShell Az module
+description: Introducing the new Azure PowerShell module Az, the replacement for the AzureRM module.
+ms.date: 10/22/2018
+author: sptramer

```

Comment:
I would remove author, ms.author, and ms.manager since these are defined in the docfx.json. I would also look at ms.topic; it could be managed in the docfx.json too.